scripts/interventions_schema.json

```diff
@@ -0,0 +1,106 @@
+{
+    "description": "",
+    "title": "",
+    "plural_title": "",
+    "definitions": {
+        "Intervention Details": {
```

Comment:
There was a PR yesterday (https://github.com/WorldBank-Transport/DRIVER/pull/377) that changed the format of these ids. The gist is that this (and its references) should change to "interventionDetails". You'll probably want to rebase and/or make the change here manually.